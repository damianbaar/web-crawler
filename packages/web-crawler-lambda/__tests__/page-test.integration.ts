import request from 'supertest'

const pageToTest = 'https://www.w3schools.com/'

test('test w3 page', () =>
  // TODO this path should comes from outside
  request('localhost:8080')
    .get(`/get-site-map?page=${pageToTest}`)
    .expect(200)
    .then(response => {
      console.log(response.body)
      expect(response.body).not.toBeNull()
      expect(response.body).toMatchSnapshot()
    }
    )
    .catch((e: Error) => fail(e))
)