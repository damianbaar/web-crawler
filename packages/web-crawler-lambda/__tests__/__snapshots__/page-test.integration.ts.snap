// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test w3 page 1`] = `
Object {
  "https://www.w3schools.com/": Array [
    Object {
      "links": Array [
        Object {
          "labels": Array [
            "w3schools",
            ".com",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "//www.w3schools.com",
        },
        Object {
          "labels": Array [
            "ïƒ‰",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "javascript:void(0)",
        },
        Object {
          "labels": Array [
            "Learn HTML",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/html/default.asp",
        },
        Object {
          "labels": Array [
            "Learn CSS",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/css/default.asp",
        },
        Object {
          "labels": Array [
            "Learn W3.CSS",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/w3css/default.asp",
        },
        Object {
          "labels": Array [
            "Learn Colors",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/colors/default.asp",
        },
        Object {
          "labels": Array [
            "Learn Bootstrap 3",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/bootstrap/default.asp",
        },
        Object {
          "labels": Array [
            "Learn Bootstrap 4",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/bootstrap4/default.asp",
        },
        Object {
          "labels": Array [
            "Learn Icons",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/icons/default.asp",
        },
        Object {
          "labels": Array [
            "Learn Graphics",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/graphics/default.asp",
        },
        Object {
          "labels": Array [
            "Learn How To",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/howto/default.asp",
        },
        Object {
          "labels": Array [
            "Learn JavaScript",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/js/default.asp",
        },
        Object {
          "labels": Array [
            "Learn jQuery",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/jquery/default.asp",
        },
        Object {
          "labels": Array [
            "Learn AngularJS",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/angular/default.asp",
        },
        Object {
          "labels": Array [
            "Learn JSON",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/js/js_json_intro.asp",
        },
        Object {
          "labels": Array [
            "Learn AJAX",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/js/js_ajax_intro.asp",
        },
        Object {
          "labels": Array [
            "Learn W3.JS",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/w3js/default.asp",
        },
        Object {
          "labels": Array [
            "Learn SQL",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/sql/default.asp",
        },
        Object {
          "labels": Array [
            "Learn PHP 5",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/php/default.asp",
        },
        Object {
          "labels": Array [
            "Learn PHP 7",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/php7/default.asp",
        },
        Object {
          "labels": Array [
            "Learn Python",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/python/default.asp",
        },
        Object {
          "labels": Array [
            "Learn Java",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/java/default.asp",
        },
        Object {
          "labels": Array [
            "Learn ASP",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/asp/default.asp",
        },
        Object {
          "labels": Array [
            "Learn Node.js",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/nodejs/default.asp",
        },
        Object {
          "labels": Array [
            "Learn Raspberry Pi",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/nodejs/nodejs_raspberrypi.asp",
        },
        Object {
          "labels": Array [
            "Web Templates",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/w3css/w3css_templates.asp",
        },
        Object {
          "labels": Array [
            "Web Statistics",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/browsers/default.asp",
        },
        Object {
          "labels": Array [
            "Web Certificates",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/cert/default.asp",
        },
        Object {
          "labels": Array [
            "Web Editor",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/tryit/default.asp",
        },
        Object {
          "labels": Array [
            "Web Development",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/whatis/default.asp",
        },
        Object {
          "labels": Array [
            "Learn XML",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/xml/default.asp",
        },
        Object {
          "labels": Array [
            "Learn XML AJAX",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/xml/ajax_intro.asp",
        },
        Object {
          "labels": Array [
            "Learn XML DOM",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/xml/dom_intro.asp",
        },
        Object {
          "labels": Array [
            "Learn XML DTD",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/xml/xml_dtd_intro.asp",
        },
        Object {
          "labels": Array [
            "Learn XML Schema",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/xml/schema_intro.asp",
        },
        Object {
          "labels": Array [
            "Learn XSLT",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/xml/xsl_intro.asp",
        },
        Object {
          "labels": Array [
            "Learn XPath",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/xml/xpath_intro.asp",
        },
        Object {
          "labels": Array [
            "Learn XQuery",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/xml/xquery_intro.asp",
        },
        Object {
          "labels": Array [
            "HTML Reference",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/tags/default.asp",
        },
        Object {
          "labels": Array [
            "CSS Reference",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/cssref/default.asp",
        },
        Object {
          "labels": Array [
            "JS Reference",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/jsref/default.asp",
        },
        Object {
          "labels": Array [
            "SQL Reference",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/sql/sql_ref_keywords.asp",
        },
        Object {
          "labels": Array [
            "PHP Reference",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/php/php_ref_overview.asp",
        },
        Object {
          "labels": Array [
            "jQuery Reference",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/jquery/jquery_ref_overview.asp",
        },
        Object {
          "labels": Array [
            "Python Reference",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/python/python_reference.asp",
        },
        Object {
          "labels": Array [
            "Bootstrap 3 Reference",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/bootstrap/bootstrap_ref_all_classes.asp",
        },
        Object {
          "labels": Array [
            "Bootstrap 4 Reference",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/bootstrap4/bootstrap_ref_all_classes.asp",
        },
        Object {
          "labels": Array [
            "W3.CSS Reference",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/w3css/w3css_references.asp",
        },
        Object {
          "labels": Array [
            "Java Reference",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/java/java_ref_keywords.asp",
        },
        Object {
          "labels": Array [
            "Angular Reference",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/angular/angular_ref_directives.asp",
        },
        Object {
          "labels": Array [
            "UTF-8 Reference",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/charsets/default.asp",
        },
        Object {
          "labels": Array [
            "HTML Exercises",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/html/html_exercises.asp",
        },
        Object {
          "labels": Array [
            "CSS Exercises",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/css/css_exercises.asp",
        },
        Object {
          "labels": Array [
            "JavaScript Exercises",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/js/js_exercises.asp",
        },
        Object {
          "labels": Array [
            "SQL Exercises",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/sql/sql_exercises.asp",
        },
        Object {
          "labels": Array [
            "PHP Exercises",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/php/php_exercises.asp",
        },
        Object {
          "labels": Array [
            "Python Exercises",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/python/python_exercises.asp",
        },
        Object {
          "labels": Array [
            "Java Exercises",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/java/java_exercises.asp",
        },
        Object {
          "labels": Array [
            "Bootstrap Exercises",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/bootstrap/bootstrap_exercises.asp",
        },
        Object {
          "labels": Array [
            "TUTORIALS",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "javascript:void(0);",
        },
        Object {
          "labels": Array [
            "Learn SVG",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/graphics/svg_intro.asp",
        },
        Object {
          "labels": Array [
            "Learn Canvas",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/graphics/canvas_intro.asp",
        },
        Object {
          "labels": Array [
            "HTML Event Reference",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/tags/ref_eventattributes.asp",
        },
        Object {
          "labels": Array [
            "HTML Attribute Reference",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/tags/ref_attributes.asp",
        },
        Object {
          "labels": Array [
            "HTML Canvas Reference",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/tags/ref_canvas.asp",
        },
        Object {
          "labels": Array [
            "HTML SVG Reference",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/graphics/svg_reference.asp",
        },
        Object {
          "labels": Array [
            "Google Maps Reference",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/graphics/google_maps_reference.asp",
        },
        Object {
          "labels": Array [
            "CSS Browser Support",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/cssref/css3_browsersupport.asp",
        },
        Object {
          "labels": Array [
            "CSS Selector Reference",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/cssref/css_selectors.asp",
        },
        Object {
          "labels": Array [
            "Icon Reference",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/icons/icons_reference.asp",
        },
        Object {
          "labels": Array [
            "W3.JS Reference",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/w3js/w3js_references.asp",
        },
        Object {
          "labels": Array [
            "ASP Reference",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/asp/asp_ref_response.asp",
        },
        Object {
          "labels": Array [
            "XML DOM Reference",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/xml/dom_nodetype.asp",
        },
        Object {
          "labels": Array [
            "XML Http Reference",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/xml/dom_http.asp",
        },
        Object {
          "labels": Array [
            "XSLT Reference",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/xml/xsl_elementref.asp",
        },
        Object {
          "labels": Array [
            "XML Schema Reference",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/xml/schema_elements_ref.asp",
        },
        Object {
          "labels": Array [
            "HTML ASCII",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/charsets/ref_html_ascii.asp",
        },
        Object {
          "labels": Array [
            "HTML ANSI",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/charsets/ref_html_ansi.asp",
        },
        Object {
          "labels": Array [
            "HTML ISO-8859-1",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/charsets/ref_html_8859.asp",
        },
        Object {
          "labels": Array [
            "HTML Symbols",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/charsets/ref_html_symbols.asp",
        },
        Object {
          "labels": Array [
            "HTML UTF-8",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/charsets/ref_html_utf8.asp",
        },
        Object {
          "labels": Array [
            "HTML Examples",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/html/html_examples.asp",
        },
        Object {
          "labels": Array [
            "CSS Examples",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/css/css_examples.asp",
        },
        Object {
          "labels": Array [
            "W3.CSS Examples",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/w3css/w3css_examples.asp",
        },
        Object {
          "labels": Array [
            "Bootstrap Examples",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/bootstrap/bootstrap_examples.asp",
        },
        Object {
          "labels": Array [
            "SVG Examples",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/graphics/svg_examples.asp",
        },
        Object {
          "labels": Array [
            "HTML Quiz",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/html/html_quiz.asp",
        },
        Object {
          "labels": Array [
            "CSS Quiz",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/css/css_quiz.asp",
        },
        Object {
          "labels": Array [
            "JavaScript Quiz",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/js/js_quiz.asp",
        },
        Object {
          "labels": Array [
            "SQL Quiz",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/sql/sql_quiz.asp",
        },
        Object {
          "labels": Array [
            "PHP Quiz",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/php/php_quiz.asp",
        },
        Object {
          "labels": Array [
            "Python Quiz",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/python/python_quiz.asp",
        },
        Object {
          "labels": Array [
            "Bootstrap Quiz",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/bootstrap/bootstrap_quiz.asp",
        },
        Object {
          "labels": Array [
            "jQuery Quiz",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/jquery/jquery_quiz.asp",
        },
        Object {
          "labels": Array [
            "XML Quiz",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/xml/xml_quiz.asp",
        },
        Object {
          "labels": Array [
            "JavaScript Examples",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/js/js_examples.asp",
        },
        Object {
          "labels": Array [
            "HTML DOM Examples",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/js/js_dom_examples.asp",
        },
        Object {
          "labels": Array [
            "jQuery Examples",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/jquery/jquery_examples.asp",
        },
        Object {
          "labels": Array [
            "AngularJS Examples",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/angular/angular_examples.asp",
        },
        Object {
          "labels": Array [
            "AJAX Examples",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/js/js_ajax_examples.asp",
        },
        Object {
          "labels": Array [
            "W3.JS Examples",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/w3js/w3js_examples.asp",
        },
        Object {
          "labels": Array [
            "PHP Examples",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/php/php_examples.asp",
        },
        Object {
          "labels": Array [
            "Python Examples",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/python/python_examples.asp",
        },
        Object {
          "labels": Array [
            "Java Examples",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/java/java_examples.asp",
        },
        Object {
          "labels": Array [
            "ASP Examples",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/asp/asp_examples.asp",
        },
        Object {
          "labels": Array [
            "XML Examples",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/xml/xml_examples.asp",
        },
        Object {
          "labels": Array [
            "XSLT Examples",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/xml/xsl_examples.asp",
        },
        Object {
          "labels": Array [
            "XPath Examples",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/xml/xpath_examples.asp",
        },
        Object {
          "labels": Array [
            "XML Schema Examples",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/xml/schema_example.asp",
        },
        Object {
          "labels": Array [
            "AJAX Examples",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/xml/ajax_examples.asp",
        },
        Object {
          "labels": Array [
            "jQuery Exercises",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/jquery/jquery_exercises.asp",
        },
        Object {
          "labels": Array [
            "Try it Yourself Â»",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/html/tryit.asp?filename=tryhtml_default",
        },
        Object {
          "labels": Array [
            "Try it Yourself Â»",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/css/tryit.asp?filename=trycss_default",
        },
        Object {
          "labels": Array [
            "Try it Yourself Â»",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/js/tryit.asp?filename=tryjs_default",
        },
        Object {
          "labels": Array [
            "cannot_determine_label",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/colors/colors_picker.asp",
        },
        Object {
          "labels": Array [
            "QUIZZES",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/quiztest/default.asp",
        },
        Object {
          "labels": Array [
            "EXERCISES",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/html/exercise.asp?filename=exercise_attributes1",
        },
        Object {
          "labels": Array [
            "FORUM",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/forum/default.asp",
        },
        Object {
          "labels": Array [
            "ABOUT",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/about/default.asp",
        },
        Object {
          "labels": Array [
            "terms of use",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/about/about_copyright.asp",
        },
        Object {
          "labels": Array [
            "cookie and privacy policy",
          ],
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/about/about_privacy.asp",
        },
        Object {
          "labels": "Colorpicker",
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "/images/colorpicker.png",
        },
        Object {
          "labels": "W3.CSS Templates",
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "w3css_templates.jpg",
        },
        Object {
          "labels": "W3 Certificates",
          "links": Array [],
          "parent": "https://www.w3schools.com/",
          "url": "getcertified.jpg",
        },
      ],
      "url": "https://www.w3schools.com/",
    },
  ],
}
`;
